kernel/packages/shared/comms/chat.ts

```diff
@@ -1,9 +1,11 @@
-import { MessageEntry, ChatMessageType } from 'shared/types'
+import { MessageEntry, ChatMessageType, PresenceStatus, UpdateUserStatusMessage } from 'shared/types'
 import { uuid } from 'atomicHelpers/math'
 import { StoreContainer } from '../store/rootTypes'
 import { messageReceived } from '../chat/actions'
+import { getProfile } from 'shared/profiles/selectors'
 
 declare const globalThis: StoreContainer
+let friendStatus: any = {}
```

Comment:
Let's improve type check here.

Reply:
Agree. This is basically a `Record<string, UpdateUserStatusMessage>` right?